setup.py

```diff
@@ -29,4 +39,10 @@
     url="https://github.com/FidelityInternational/djangocms-pageadmin",
     license="BSD",
     test_suite="tests.settings.run",
+    tests_require=TESTS_REQUIRE,
```

Comment:
I'll need to run this and see how we were running the test suite before. I think most of these packages where tested as parts of the roadbuster project before. They should be self contained though so these changes are probably required so this would be the right thing to do.